fault/actions.py

```diff
@@ -22,14 +25,23 @@ def __init__(self, port, value):
         self.port = port
         self.value = value
 
+    def __str__(self):
+        return f"Expect({self.port.debug_name}, {self.value})"
+
 
 class Eval(Action):
     def __init__(self):
         super().__init__()
 
+    def __str__(self):
+        return f"Eval()"
+
 
 class Step(Action):
     def __init__(self, steps, clock):
         super().__init__()
         self.steps = steps
         self.clock = clock
+
+    def __str__(self):
+        return f"Step({self.steps}, {self.clock.debug_name})"
```

Comment:
maybe `Step({self.clock.debug_name}, steps={self.steps})` to make it more clear? Idk how standard the 'steps' argument is.

Reply:
Done